Test/regressions/InitOutsideCtors.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.4.24;
+
+contract A {
+    uint a=1;
+    uint b;
+    bool c=true;

```

Comment:
Is also getting strings in too difficult? Or do it it another PR?

Reply:
It looks like string is handled. can you add a couple of lines in this example along with asserts?